ocaml/xapi/xapi_vm_snapshot.ml

```diff
@@ -289,9 +299,19 @@ let update_vifs_vbds_and_vgpus ~__context ~snapshot ~vm =
 
 		debug "Cloning the snapshoted disks";
 		let driver_params = Xapi_vm_clone.make_driver_params () in
-		let cloned_disks = Xapi_vm_clone.safe_clone_disks rpc session_id Xapi_vm_clone.Disk_op_clone ~__context snap_vbds driver_params in
+		let cloned_disks = Xapi_vm_clone.safe_clone_disks rpc session_id Xapi_vm_clone.Disk_op_clone ~__context snap_vbds_without_cd driver_params in
 		TaskHelper.set_progress ~__context 0.5;
 
+		debug "Updating the snapshot_of fields for relevant VDIs";
+		let original_to_newvdi_map = List.map2 (fun oldvdi (_, newvdi, _) -> (oldvdi,newvdi)) vdis_snap_of cloned_disks in
```

Comment:
Renaming `original_to_newvdi_map` to `oldvdi_to_newvdi_map` would be more consistent.